content/developer/backend/server-apache.md

```diff
@@ -31,3 +31,11 @@ The provided `.htaccess` already comes with some decent hardening:
 
 - The only php script that can be requested directly is `/index.php`.
 - Folders that should not be accessed from the web return a 403.
+
+### Default VirtualHost entry
+
+By default, Apache will respond to any ServerName option until a domain which doesn't match any VirtualHost is requested,
+When this happens, the first VirtualHost is used as a catch all.
+Make sure you have a default entry added to mitigate host-header-injection type of attacks.
```

Comment:
Let's make this final sentence more specific ("mitigate" is industry slang and not super clear to average users, and "host header attack" bears a bit of elaboration).

> Make sure you have a default entry added. Not doing so can make you susceptible to host header injection attacks (tricking your server into rendering pages based on a third-party domain).